setup.cfg

```diff
@@ -22,7 +22,8 @@ data_files =
     /usr/share/ansible/roles/ipareplica = roles/ipareplica/*
 
 [flake8]
-extend-ignore = E203
+extend-ignore = E203,D100,D101,D102,D103,D105,D107
```

Comment:
I think that by default flake8 doesn't look at docstrings. Why do you need that? If we are using flake8-docstrings we don't need pydocstyle and vice-versa.

Reply:
Also you should either ignore (D212 or D213) and (D202 or D203).

Reply:
Added almost the same configuration to flake8. I needed to add D400 and D401 to flake8 to obtain the same result as pydocstyle.